pkg/actors/internal_actor_test.go

```diff
@@ -159,11 +160,10 @@ func TestInternalActorCall(t *testing.T) {
 		// Verify the response metadata matches what we expect
 		assert.Equal(t, int32(200), resp.Status().Code)
 		contentType := resp.ContentType()
-		data, _ := resp.RawDataFull()
 		assert.Equal(t, invokev1.OctetStreamContentType, contentType)
 
 		// Verify the actor got all the expected inputs (which are echoed back to us)
-		info, err := decodeTestResponse(data)
+		info, err := decodeTestResponse(resp.RawData())
```

Comment:
What was the purpose behind changing this from RawDataFull() to RawData()?

Reply:
See https://github.com/dapr/dapr/pull/6819#discussion_r1296555262